modules/era/sql/era_droplist.sql

```diff
@@ -6255,11 +6258,11 @@ INSERT INTO `mob_droplist` VALUES (737,0,0,1000,18944,20);       -- Death Sickle
 INSERT INTO `mob_droplist` VALUES (737,0,0,1000,19102,20);       -- Main Gauche (2.0%)
 
 -- ZoneID: 204 - Eastern Shadow
-INSERT INTO `mob_droplist` VALUES (738,0,0,1000,940,@ALWAYS); -- Revival Tree Root (Always, 100%)
-INSERT INTO `mob_droplist` VALUES (738,0,0,1000,18714,@RARE); -- Valis Bow (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (738,0,0,1000,940,@ALWAYS);     -- Revival Tree Root (Always, 100%)
+INSERT INTO `mob_droplist` VALUES (738,0,0,1000,18714,@UNCOMMON); -- Valis Bow (Uncommon, 10%)
```

Comment:
I would keep this a 5% according to the data.
FFXIDB: TH0 only has 14 points @ 35.7%, TH3+ @ 44.8%
FFXIWiki: 717 Points at 9.5% which leads me to believe (assuming this NM was killed with at least TH3+) would still be 5%.